src/components/accounts/CreateAccount.js

```diff
@@ -316,6 +318,16 @@ class CreateAccount extends Component {
                             onChange={this.handleInput} />
                         {passwordConfirmationWarning}
                     </FormGroup>
+                    <FormGroup bsClass={bioClass}>
+                        <Label>Bio</Label>
+                        <input type='bio'
```

Comment:
Please use `<textarea> `instead of `<input>` as the bio can be quite longer than normal input

Reply:
I second this, good call @kevinyok.

Reply:
Ok, I changed it